src/features/filter-transactions/model.ts

```diff
@@ -1,2 +1,21 @@
-export type Status = 'committed' | 'rejected' | null;
-export type Tab = 'all' | 'transfers' | 'mints' | 'burns' | 'grants' | 'revokes';
+export type TransactionStatus = 'committed' | 'rejected' | null;
+export type TransactionTypeTabs = 'all' | 'transfers' | 'mints' | 'burns' | 'grants' | 'revokes';
+export type DefaultTransactionTypeTabs = 'transactions' | 'transfers' | 'mints' | 'burns' | 'grants' | 'revokes';
+
+export const defaultTransactionTypeOptions = [
+  { label: 'transactions', value: 'transactions' },
+  { label: 'transfers', value: 'transfers' },
+  { label: 'mints', value: 'mints' },
+  { label: 'burns', value: 'burns' },
+  { label: 'grants', value: 'grants' },
+  { label: 'revokes', value: 'revokes' },
+];
+
+export const transactionTypeOptions = [
+  { label: 'all', value: 'all' },
+  { label: 'transfers', value: 'transfers' },
+  { label: 'mints', value: 'mints' },
+  { label: 'burns', value: 'burns' },
+  { label: 'grants', value: 'grants' },
+  { label: 'revokes', value: 'revokes' },
+];
```

Comment:
lets keep it as is. I will rename it to better understanding. Maybe later we will extend/reduce those filters. They are similar but different entities. Also i18n isn't lost since we use them only in BaseTabs component that has the following label display:
```
{{ $t(item.label) }}
```

Reply:
Okay, let's keep it.

> `{{ $t(item.label) }}`

Maybe `i18nKey` instead of `label` would be clearer.

Reply:
Okay, let's keep it.

> `{{ $t(item.label) }}`

Maybe `i18nKey` instead of `label` would be clearer.